utils_linux.go

```diff
@@ -240,7 +241,15 @@ func (r *runner) run(config *specs.Process) (int, error) {
 			return -1, err
 		}
 	}
-	if r.detach {
+	if !r.create {
+		if err := process.Signal(syscall.SIGCONT); err != nil {
+			r.terminate(process)
+			r.destroy()
+			tty.Close()
+			return -1, err
+		}
+	}
+	if r.detach || r.create {
```

Comment:
On Tue, May 17, 2016 at 01:36:38PM -0700, Michael Crosby wrote:
> 	if err := osutils.SetSubreaper(1); err != nil {

This is fairly low-level for “I'd like to collect the container's exit
code”.  I think callers should at least have the option of having the
‘create’ process stay attached through container death.

That also allows the create process to do psuedo-terminal copying [1],
although the OCI spec around process.terminal is currently not very
clear to me [2].

[1]: https://github.com/wking/ccon/tree/create-start#terminal
[2]: https://groups.google.com/a/opencontainers.org/forum/#!topic/dev/hcbwL4T9lGs
     Subject: More detail for process.terminal?
     Date: Thu, 14 Jan 2016 14:44:08 -0800
     Message-ID: <20160114224408.GM6362@odin.tremily.us>